R.swift/input.swift

```diff
@@ -177,6 +186,12 @@ struct CallInformation {
 
       productModuleName = try getFirstArgumentForOption(productModuleNameOption, environment["PRODUCT_MODULE_NAME"])
 
+      if let rswiftignorePath = options[rswiftignore]?.first {
+        rswiftignoreURL = URL(fileURLWithPath: rswiftignorePath)
+      } else {
+        rswiftignoreURL = nil
+      }
```

Comment:
This fallback is exactly what `getFirstArgumentForOption` will do right? So I don't see why we shouldn't use that here.